### DISCUSSION TITLE
Suppress <p> tags

### DISCUSSION THREAD

**Discussion by bayre** [markdoc/markdoc#79] [Q&A] [answered]
Is it possible to prevent the child content of a custom tag from rendering within a `<p>` tag? I have tried modifying the `children` [option](https://markdoc.io/docs/tags#options) but it looks as though that only affects validation.

p.s. thanks for markdoc! 🙏

**Comment by mfix-stripe** [comment, marked as answer]
@bayre you can customize the `transform` option to change what will ultimately get rendered: https://markdoc.io/docs/tags#options

Please note: if you use an ["inline tag"](https://markdoc.io/docs/syntax#tags): 

```js
{% inline-tag %} Inline {% /inline-tag %}
```

this tag will get wrapped in a `p` node. See https://markdoc.io/docs/syntax#tags for context.

**Reply by KingOfTac** [reply]
I am running into this as well. It would be nice to be able to disable the default behavior without needing to explicitly create new lines.

This would be a useful ergonomic feature for cases where you have custom tags built on web components that are using slots which are looking for specific tag names for styling via the `::slotted` selector.
